tests/test_ssl/test_with_ssl.py

```diff
@@ -0,0 +1,150 @@
+import json
+import time
+import os
+
+import mrsal.config.config as config
+import pika
+# import tests.config as test_config
+from mrsal.config.logging import get_logger
+from mrsal.mrsal import Mrsal
+
+log = get_logger(__name__)
+
+host = os.environ.get('RABBITMQ_DOMAIN_TLS')
+port = os.environ.get('RABBITMQ_PORT_TLS')
+credentials = (os.environ.get('RABBITMQ_DEFAULT_USER'), os.environ.get('RABBITMQ_DEFAULT_PASS'))
+virtual_host = os.environ.get('RABBITMQ_DEFAULT_VHOST')
+
+mrsal = Mrsal(host=host,
+              port=port,
+              credentials=credentials,
+              virtual_host=virtual_host,
+              ssl=True)
```

Comment:
you need to add in docs how to test this. We can't publish our credentials or certificates. Let the user know that they need to give the path to the TLS certs they have created and if they don't do this then SSL setup will fail.